.github/workflows/goreleaser.yaml

```diff
@@ -3,7 +3,7 @@ name: goreleaser
 on:
   push:
     tags:
-      - 'v*.*.*'
+      - '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
am a bit confused with this change; is this changing the tagging to drop the `v` ?

Reply:
oh whoops
```suggestion
      - 'v[0-9]+.[0-9]+.[0-9]+'
```